piff/gp_interp_2pcf.py

```diff
@@ -253,6 +275,20 @@ def initialize(self, stars, logger=None):
                 self.kernels = [copy.deepcopy(ker) for ker in self.kernel_template]                                        
         return stars
 
+    def _build_average(self, X):
```

Comment:
for readability it might help to name this something that mentions it is an average from _prior_ information. I initially read this and was confused because it reads as being an average from the input data.